packages/moralis/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@uauth/moralis",
+  "version": "1.0.0",
+  "source": "./src/index.ts",
+  "main": "./build/index.cjs",
+  "module": "./build/index.module.mjs",
+  "unpkg": "./build/index.umd.js",
+  "browser": "./build/index.umd.js",
+  "exports": {
+    "import": "./build/index.module.mjs",
+    "require": "./build/index.cjs"
+  },
+  "type": "module",
+  "types": "./build/index.d.ts",
+  "files": [
+    "build"
+  ],
+  "amdName": "UAuthMoralis",
+  "scripts": {
+    "build": "microbundle --jsx 'React.createElement'",
+    "dev": "microbundle watch --jsx 'React.createElement'",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
```

Comment:
You should also figure out what versions actually will work with the package. Instead of just the latest version. We want to be as flexible as with our package versions. In the past a lot integrations have used out-of-date versions of libraries that we integrate with.